x-pack/plugins/observability_solution/infra/public/alerting/metric_threshold/components/groups.tsx

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+
+export function Groups({ groups }: { groups: Array<{ field: string; value: string }> }) {
```

Comment:
I've added this Groups component temporarily until we share the Source component between different rules: https://github.com/elastic/kibana/issues/181692